src/screens/Login/index.js

```diff
@@ -0,0 +1,101 @@
+import React, {useState, useEffect} from 'react';
+import {
+  ActivityIndicator,
+  Keyboard,
+  KeyboardAvoidingView,
+  Alert,
+} from 'react-native';
+import {Block, TextView, Input, Button} from '../../components';
+import s from './styles';
+import {theme} from '../../constants';
+
+const VALID_EMAIL = 'thienphu';
+const VALID_PASSWORD = '123456';
```

Comment:
Should not display pass on code. You have 2 options:
Option 1: You can encrypt this pass
Option 2: Disable the login page, currently we don't need login.

Reply:
I choose option 2